sysdig/data_source_sysdig_fargate_ECS_test.go

```diff
@@ -164,13 +171,21 @@ func TestNewPatchOptions(t *testing.T) {
 			"stream_prefix": "fried",
 			"region":        "chicken",
 		},
-		Essential: true,
+		Essential: false,
 	}
 	actualPatchOptions := newPatchOptions(data)
 
 	if !reflect.DeepEqual(expectedPatchOptions, actualPatchOptions) {
 		t.Errorf("patcConfigurations are not equal. Expected: %v, Actual: %v", expectedPatchOptions, actualPatchOptions)
 	}
+
+	data.Set("priority", "security")
```

Comment:
Sadly this error is ours - there are some copy/pasta opportunities.

Reply:
oh. good catch. I haven't seen it complain about this line, must have somehow missed it